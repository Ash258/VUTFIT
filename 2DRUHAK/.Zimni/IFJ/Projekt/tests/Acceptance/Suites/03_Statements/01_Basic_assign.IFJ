' Basic constant assignment

scope
	dim a as string
	a =    !"fooman"

	dim b as integer
	b =  42

	dim c as double
	c  =  74.85

	print a; b; c;
end scope
