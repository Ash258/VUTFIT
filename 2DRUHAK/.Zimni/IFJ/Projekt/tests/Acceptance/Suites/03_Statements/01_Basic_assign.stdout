fooman 42 74.85